packages/app-studio-toolkit/src/utils/bas-utils.ts

```diff
@@ -12,7 +18,7 @@ export enum ExtensionRunMode {
   unexpected = `unexpected`,
 }
 
-export function shouldRunCtlServer(): boolean {
+export function isBAS(): boolean {
```

Comment:
The function name is now confusing. it returns true in the case of "BAS", "hybrid" and "PE". the previous name was more meaningful